var/spack/repos/builtin/packages/py-tensorflow-probability/package.py

```diff
@@ -33,20 +34,23 @@ class PyTensorflowProbability(Package):
     depends_on("py-wheel", type="build")
     depends_on("py-setuptools", type="build")
 
-    depends_on("py-six@1.10.0:", type=("build", "run"))
+    # required_packages.py
+    depends_on("py-absl-py", when="@0.18:", type=("build", "run"))
+    depends_on("py-six@1.10:", type=("build", "run"))
     depends_on("py-numpy@1.13.3:", type=("build", "run"))
     depends_on("py-decorator", type=("build", "run"))
-
-    depends_on("py-tensorflow@1.14:", type=("build", "run"), when="@0.8.0")
-    depends_on("py-gast@0.2.0:0.2", type=("build", "run"), when="@0.8.0")
-    depends_on("py-cloudpickle@1.1.1", type=("build", "run"), when="@0.8.0")
-
-    depends_on("py-tensorflow@2.4:", type=("build", "run"), when="@0.12.0:")
-    depends_on("py-cloudpickle@1.3:", type=("build", "run"), when="@0.12.0:")
-    depends_on("py-gast@0.3.2:", type=("build", "run"), when="@0.12.0:")
-    depends_on("py-dm-tree", type=("build", "run"), when="@0.12.0:")
-
-    depends_on("bazel@3.2.0:", type="build")
+    depends_on("py-cloudpickle@1.3:", when="@0.12:", type=("build", "run"))
+    depends_on("py-cloudpickle@1.1.1", when="@0.8", type=("build", "run"))
+    depends_on("py-gast@0.3.2:", when="@0.12:", type=("build", "run"))
+    depends_on("py-gast@0.2", when="@0.8", type=("build", "run"))
+    depends_on("py-dm-tree", when="@0.12:", type=("build", "run"))
+
+    # tensorflow_probability/python/__init__.py
```

Comment:
Do you want to leave this comment here?

Reply:
Yes, this helps future me remember where I found these dependency versions since they aren't documented in the usual locations.

Reply:
I inferred that was likely the intension though I wonder if a little more would be useful, such as at least `source: tensorflow_probability/python/__init__.py` so the non-Adam person who comes along has a better chance of understanding the comment.